src/Sentry.Unity/UnityWebRequestTransport.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Sentry.Extensibility;
+using Sentry.Http;
+using Sentry.Protocol.Envelopes;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Sentry.Unity
+{
+    internal class WebBackgroundWorker : IBackgroundWorker
+    {
+        private readonly SentryMonoBehaviour _behaviour;
+        private readonly UnityWebRequestTransport _transport;
+
+        public WebBackgroundWorker(SentryUnityOptions options, SentryMonoBehaviour behaviour)
+        {
+            _behaviour = behaviour;
+            _transport = new UnityWebRequestTransport(options);
+        }
+
+        public bool EnqueueEnvelope(Envelope envelope)
+        {
+            _ = _behaviour.StartCoroutine(_transport.SendEnvelopeAsync(envelope));
+            return true;
+        }
+
+        public Task FlushAsync(TimeSpan timeout) => Task.CompletedTask;
+
+        public int QueuedItems { get; }
+    }
+
+    internal class UnityWebRequestTransport : HttpTransportBase
+    {
+        private readonly SentryUnityOptions _options;
+
+        public UnityWebRequestTransport(SentryUnityOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        // adapted HttpTransport.SendEnvelopeAsync()
+        internal IEnumerator SendEnvelopeAsync(Envelope envelope)
+        {
+            using var processedEnvelope = ProcessEnvelope(envelope);
+            if (processedEnvelope.Items.Count > 0)
+            {
+                // Send envelope to ingress
+                var httpRequest = CreateRequest(processedEnvelope);
+                var www = CreateWebRequest(httpRequest);
+                yield return www.SendWebRequest();
+
+                var response = GetResponse(www);
+                if (response != null)
+                {
+                    HandleResponse(response, processedEnvelope);
+                }
```

Comment:
When can it be null? Should we log?

```suggestion
                if (response is not null)
                {
                    HandleResponse(response, processedEnvelope);
                } else {
                  // log?
                }
```

Reply:
it's already logged when null (i.e. when there was a network error)